app/controllers/find_commodities_controller.rb

```diff
@@ -1,5 +1,5 @@
 class FindCommoditiesController < ApplicationController
-  before_action :disable_switch_service_banner, only: [:show]
+  before_action :disable_switch_service_banner!, only: [:show]
```

Comment:
I am happy to use the bang `!` notation, but we haven't used it for the other `disable_...` methods,
should we change it for the others  too?